frappe/utils/install.py

```diff
@@ -144,40 +148,80 @@ def import_country_and_currency():
 
 	data = get_all()
 
+	print("\nOverwriting your existing Country & Currency data in 10 seconds, press Enter to abort...")
```

Comment:
This seems unusual. If we want to give the choice there should at least be a command flag to toggle this option for unattended installations.

Reply:
And how would people know to use the flag?

Reply:
Most of the cases this will be run on `bench migrate`, I guess? So `bench migrate --help` could offer `--update-country-data` or `--keep-country-data`. Or we decide that it should always/never be updated. But nobody is going to sit in front of the console waiting for the countdown to show up.

Reply:
"Nobody is going to sit in front of the console waiting..." 
So you think that they are going to run `bench migrate` automated? The prompt comes up almost immediately.
And more to the point you think they're going to look up the help on a command that everyone already knows?

Reply:
> So you think that they are going to run bench migrate automated? 

Yes, think about Frappe Cloud, for example. When somebody clicks "Update to newest Release" for their site, `bench migrate` gets executed with nobody watching.